### PR TITLE
CONTRIBUTING: add notes about placeholders and some examples

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -59,14 +59,26 @@ As a quick reference, the format of each page should match the following templat
...
+`command --option`
 
 - Example description:
 
-`command -opt1 -opt2 -arg1 {{arg_value}}`
+`command --option1 --option2 {{arg_value}}`
 ```
 
-For more detailed page formatting guidelines,
+Examples:
```
Comment (CleanMachine1) [inline CONTRIBUTING.md:69]:
```suggestion
To see some examples of preexisting pages, you can look at:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -59,14 +59,26 @@ As a quick reference, the format of each page should match the following templat
...
+
+- [pwd](https://github.com/tldr-pages/tldr/blob/main/pages/common/pwd.md) - one of the simpliest command examples
+- [caja](https://github.com/tldr-pages/tldr/blob/main/pages/linux/caja.md) - page with placeholders (read notes below)
+
+Everything inside doubled curly braces and them itself called placeholders. We use them to highlight places in `tldr` pages
```
Comment (CleanMachine1) [inline CONTRIBUTING.md:74]:
```suggestion
In our pages, we use placeholders which are defined as being things within curly brackets, for example `sleep {{5}}`, in this case the user can change 5 to any number.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -59,14 +59,26 @@ As a quick reference, the format of each page should match the following templat
...
+where user can write arbitrary arguments such as directories, numbers, .etc. The most common placeholders are:
```
Comment (CleanMachine1) [inline CONTRIBUTING.md:75]:
```suggestion
Other examples but not limited to of our placeholder syntax are:
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -59,14 +59,26 @@ As a quick reference, the format of each page should match the following templat
...
+
+- `{{path/to/directory}}` - just one directory
```
Comment (CleanMachine1) [inline CONTRIBUTING.md:77]:
```suggestion
- `{{path/to/directory}}`
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -59,14 +59,26 @@ As a quick reference, the format of each page should match the following templat
...
+- `{{path/to/directory1 path/to/directory2 ...}}` - any number of directories. We don't not mention anything about how many directories
```
Comment (CleanMachine1) [inline CONTRIBUTING.md:78]:
```suggestion
- `{{path/to/directory1 path/to/directory2 ...}}`
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -59,14 +59,26 @@ As a quick reference, the format of each page should match the following templat
...
+  user have to pass to some command.
```
Comment (CleanMachine1) [inline CONTRIBUTING.md:79]:
```suggestion
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -59,14 +59,26 @@ As a quick reference, the format of each page should match the following templat
...
 
-`command -opt1 -opt2 -arg1 {{arg_value}}`
+`command --option1 --option2 {{arg_value}}`
 ```
 
-For more detailed page formatting guidelines,
+Examples:
+
+- [pwd](https://github.com/tldr-pages/tldr/blob/main/pages/common/pwd.md) - one of the simpliest command examples
+- [caja](https://github.com/tldr-pages/tldr/blob/main/pages/linux/caja.md) - page with placeholders (read notes below)
```
Comment (CleanMachine1) [inline CONTRIBUTING.md:72]:
```suggestion
- [caja](https://github.com/tldr-pages/tldr/blob/main/pages/common/tar.md) - page with placeholders 
```

**Review by CleanMachine1 — COMMENTED** [review]
Not sure if I agree with this addition to the page in the first place, however here are some suggestions I think make it more readable.

**Comment by EmilySeville7cfg** [comment]
@CleanMachine1, thanks for suggestions! 👍🏻 Please review #8364 and tell me whether it's better to add a separate tutorial instead of bloating up our CONTRIBUTING.md. This PR can be declined with respect to #8364 one.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -59,14 +59,25 @@ As a quick reference, the format of each page should match the following templat
...
+`command --option1 --option2 {{arg_value}}`
 ```
 
-For more detailed page formatting guidelines,
+To see some examples of preexisting pages, you can look at:
+
+- [pwd](https://github.com/tldr-pages/tldr/blob/main/pages/common/pwd.md) - one of the simpliest command examples
+- [caja](https://github.com/tldr-pages/tldr/blob/main/pages/common/tar.md) - page with placeholders 
+
+In our pages, we use placeholders which are defined as being things within curly brackets, for example `sleep {{5}}`, in this case the user can change 5 to any number.
```
Comment (mfrw) [inline CONTRIBUTING.md:74]: How about we use `token` here ?
```suggestion
In our pages, we use placeholders which are defined as being tokens within curly brackets, for example `sleep {{5}}`, in this case the user can change 5 to any number.
```

**Review by mfrw — APPROVED** [review]
LGTM 🥇

**Review by blueskyson — APPROVED** [review]
Look good to me.

**Pull request merged**